packages/mux-player-react/src/themes/news-theme.tsx

```diff
@@ -0,0 +1,593 @@
+import 'media-chrome/react';
+import 'media-chrome/react/menu';
+import React from 'react';
+
+export default function NewsTheme() {
```

Comment:
issue(blocking): This bit of refactor slipped through the cracks. From a quick scan at least, it looks like everything new in here is ads specific and not e.g. playlist related (which looks like it's all managed in the jsx/react side for playlist player). We should probably just pull this out and make it like our other html templates. Also move it to `mux-player` package and use it as default for `mux-player` ads player theme.

Reply:
we can circle back around and discuss if you want to try to break this effort up per "shortest path to removing one-off branch"/releases